src/main/java/org/opentripplanner/middleware/i18n/Message.java

```diff
@@ -19,7 +19,10 @@ public enum Message {
     ACCEPT_DEPENDENT_EMAIL_SUBJECT,
     ACCEPT_DEPENDENT_EMAIL_MANAGE,
     ACCEPT_DEPENDENT_ERROR,
+    DESTINATION_CHANGE_NOTIFICATION,
     LABEL_AND_CONTENT,
+    MODE_CHANGE_NOTIFICATION,
+    ORIGIN_CHANGE_NOTIFICATION,
```

Comment:
Rename to something like `DEPARTED_NOTIFICATION` or `LEAVE_ORIGIN_NOTIFICATION`?

Reply:
https://github.com/ibi-group/otp-middleware/pull/275/commits/91d66190cb611283993b8e745148641efd695c5e